content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,300 @@
+# Running Experiments
+
+We explain how DVC codifies and executes experiments, setting their parameters,
+using multiple jobs to run them in parallel, and running them in queues, among
+other details.
+
+> 📖 If this is the first time you are introduced into data science
+> experimentation, you may want to check the basics in
+> [Get Started: Experiments](/doc/start/experiments/) first.
+
+## The pipeline
+
+DVC relies on <abbr>pipelines</abbr> that codify experiment workflows (code,
+<abbr>stages</abbr>, <abbr>parameters</abbr>, <abbr>outputs</abbr>, etc.) in a
+`dvc.yaml` file. These contain the commands to run the experiments.
+
+> 📖 See [Get Started: Data Pipelines](/doc/start/data-pipeline) for an intro to
```

Comment:
Is this something that exists or will exist before this PR is merged?

Reply:
Exists: https://dvc.org/doc/start/data-pipelines